creator-node/src/apiHelpers.ts

```diff
@@ -135,13 +141,37 @@ export const sendResponse = (
   res.status(resp.statusCode).send(resp.object)
 }
 
+/**
+ * Wrapper helper method to send response with metrics involved
+ * @param {Object} req Express request object
+ * @param {Object} res Express response object
+ * @param {Object} routeResponse error or success response object
+ * @param {Object} prometheusResult prometheus results to track
+ */
+module.exports.sendResponseWithMetric = (
```

Comment:
can you make this 
```
export const sendResponseWithMetrics = ...
```

I think we're trying to distance ourselves from `module.exports` for typescript files